src/api/v1beta1/properties.go

```diff
@@ -235,6 +237,27 @@ func (dk *DynaKube) CommunicationHosts() []dtclient.CommunicationHost {
 	return communicationHosts
 }
 
+func TenantUUID(apiUrl string) (string, error) {
```

Comment:
IMO this should be a method of the `DynaKube`

its only used at https://github.com/Dynatrace/dynatrace-operator/pull/313/files#diff-90a82428e1d8d04b7f796e640b56423f6dac95f4554d87894943b38d5bad0329R102
and AFAIK `StsProperties` is a `Dynakube` (amoung other things 😅  )
```
type statefulSetProperties struct {
	*dynatracev1beta1.DynaKube
	*dynatracev1beta1.CapabilityProperties
	customPropertiesHash    string
	kubeSystemUID           types.UID
	feature                 string
	capabilityName          string
	serviceAccountOwner     string
	OnAfterCreateListener   []events.StatefulSetEvent
	initContainersTemplates []corev1.Container
	containerVolumeMounts   []corev1.VolumeMount
	volumes                 []corev1.Volume
}
```

Reply:
Makes sense, will do 👍